perl/README.md

```diff
@@ -0,0 +1,1025 @@
+# Bystro High Dimensional Genomic Annotator Documentation
+
+## What is the Bystro Annotator?
+
+In order to make use of genetic data, which is composed of variants (colloquially known as mutations) and the individuals (samples) that have those variants, we need to clean that data and then carefully describe those variants with as much accurate information as we can. These descriptions are known as labels, model parameters, features, or "annotations", depending on your field. These activites are collectively called data curation and data labeling, but in genetics we call them qc and annotation.
+
+The Bystro Annotator is the fastest and most comprehensive data curation and labeling library in the world for genetic data. It takes 1 or more VCF ([Variant Call Format](https://samtools.github.io/hts-specs/VCFv4.2.pdf)) or SNP ([PEMapper/PECaller](https://www.pnas.org/doi/full/10.1073/pnas.1618065114)) files as input, and outputs a cleaned and thoroughly labeled (annotated) representation of the data, along with a genotype dosage matrix in the [Arrow Feather V2/IPC format](https://arrow.apache.org/docs/python/feather.html), as well as a set of statistics that describe sample-level characteristics of the data.
+
+Bystro Annotator annotates both variant and sample genotypes. It is capable of processing millions of samples and billions of mutations on commodity hardware such as a laptop or a workstation. It is roughly **100,000** times faster than [Variant Effect Predictor](https://www.ensembl.org/info/docs/tools/vep/index.html) (VEP), **100** times faster than [Annovar](https://doc-openbio.readthedocs.io/projects/annovar/en/latest/), and **50** times faster than [Illumina Connected Annotations](https://developer.illumina.com/illumina-connected-annotations), all while outputting more annotations than any of these tools. What takes VEP years to do, Bystro can do in minutes to hours, all without requiring multiple servers.
+
+Bystro's performance isn't just about speed, it's also about comprehensiveness. In statistics "dimensionality" refers to how many parameters/covariates/descriptions a model has. Imagine that we are modeling a genetic variant to try to understand how it impacts a disease. The parameters in the model are the annotations we have for that variant. Bystro can output thousands of annotations for each variant, and can do so for millions of variants in seconds. This is why we call it "high dimensional".
+
+For exampole, Bystro can afford to provide complete annotations from gnomad v4.1, for all gnomAD populations and subpopulations, from the exomes, genomes, and joint datasets, genome-wide. This means intersecting terabytes of data over the entire genome with each individual variant, all done in microseconds per variant. **No other tool can do this**.
```

Comment:
Recommend to be a little more specific than 'this' at the end. Because the last thing you talked about was how fast it was, it sounds like the other tools can still handle terabytes of data but it just takes longer. What is the limit of the rest of the tools in terms of comprehensiveness of annotations?

Reply:
It's a bit of marketing, but broadly no tool can do any of that

Reply:
How about adding 'any of'? I do think that it might be impactful to add the limits of other annotations in terms of how much they can annotate if it's specific number of fields or amount of data (versus terabytes).
```suggestion
For example, Bystro can afford to provide complete annotations from gnomad v4.1, for all gnomAD populations and subpopulations, from the exomes, genomes, and joint datasets, genome-wide. This means intersecting terabytes of data over the entire genome with each individual variant, all done in microseconds per variant. **No other tool can do any of this**.
```